src/throttle/ratelimit.py

```diff
@@ -2,6 +2,32 @@
 from rest_framework.throttling import SimpleRateThrottle
 from django.contrib.auth.models import User
 
+# WIP: check again later
+class BaseThrottle:
+    """
+    Base rate throttling of requests.
+    """
+
+    def allow_request(self, request, view):
+        raise NotImplementedError('Requested for throttling must be set')
```

Comment:
Should be referencing into allow request method, not method for base throttle

Reply:
And also, please write testing for your changes if you don't mind @sodrooome

Reply:
Resolving with 3e4cabd

Reply:
Alright, it's good to go.

What about the test?

Reply:
I'm gonna send another pull request about that over here, since this repo doesn't have a directory that contains a single test